src/EditorFeatures/CSharpTest/ConvertNamespace/ConvertToFileScopedNamespaceAnalyzerTests.cs

```diff
@@ -737,14 +737,16 @@ class C
     {
         void M()
         {
-            System.Console.WriteLine(""""""
```

Comment:
nit: Just curious, do we know if the BCL is planning to add some Console APIs for `ROS<byte>`?

Reply:
Console's APIs are in terms of text, e.g. string.  Adding bytes-based APIs would require knowing what encoding to use to convert those bytes to text.  It may not be the same encoding of the underlying stream.   If you want to work in terms of bytes, you can always get a Stream and write to that, at which point you're saying you know your bytes are appropriate for the underlying encoding, e.g.
```C#
using Stream s = Console.OpenStandardOutput();
s.Write("hello world"u8);
```